src/UiPath.Workflow.Runtime/Runtime/CompletionCallbackWrapper.cs

```diff
@@ -35,7 +35,7 @@ internal bool SerializedNeedsToGatherOutputs
         set => _needsToGatherOutputs = value;
     }
 
-    public void CheckForCancelation() => _checkForCancelation = true;
+    internal void CheckForCancelation() => _checkForCancelation = true;
```

Comment:
changes to this file are no longer needed

Reply:
Fixed.